src/OrchardCore.Modules/OrchardCore.Media/Assets/js/fontawesome-icons-thumbnails/font-awesome-thumbnails.js

```diff
@@ -0,0 +1,68 @@
+﻿// https://github.com/spatie/font-awesome-filetypes
+
+const fa_icons = {
+    image: 'fa-regular fa-image',
+    pdf: 'fa-regular fa-file-pdf',
+    word: 'fa-regular fa-file-word',
+    powerpoint: 'fa-regular fa-file-powerpoint',
+    excel: 'fa-regular fa-file-excel',
+    csv: 'fa-regular fa-file',
+    audio: 'fa-regular fa-file-audio',
+    video: 'fa-regular fa-file-video',
+    archive: 'fa-regular fa-file-zipper',
+    code: 'fa-regular fa-file-code',
+    text: 'fa-regular fa-file-lines',
+    file: 'fa-regular fa-file'
+};
+
+const fa_thumbnails = {
+    gif: fa_icons.image,
+    jpeg: fa_icons.image,
+    jpg: fa_icons.image,
+    png: fa_icons.image,
+
```

Comment:
either remove the additional spaces or add a comment in the space.

Reply:
done